components/mediation/data-publishers/org.wso2.micro.integrator.observability/src/main/java/org/wso2/micro/integrator/obsrvability/handler/util/MetricConstants.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.micro.integrator.obsrvability.handler.util;
+
+/**
+ * Constants required in instrumenting Prometheus metrics.
+ */
+public class MetricConstants {
+
+    // Constants for Synapse artifacts
+    public static final String PROXY_NAME = "proxy.name";
+    public static final String SYNAPSE_REST_API = "SYNAPSE_REST_API";
+    public static final String INBOUND_ENDPOINT = "inbound-endpoint";
+    public static final String INBOUND_ENDPOINT_NAME = "inbound.endpoint.name";
```

Comment:
Can we remove these, and use synapse constants as much as possible?

Reply:
+1

Reply:
Fixed